src/ui/Views/Shared/_Alerts.cshtml

```diff
@@ -7,12 +7,12 @@
 @if (type != null)
 {
   <div class="govuk-@type-summary" role="alert">
-    <h3 class="govuk-heading-m govuk-@type-summary__title">@title</h3>
+    <h3 class="govuk-@type-summary__title">@title</h3>
 
     @if (body != null)
     {
       <div class="govuk-@type-summary__body">
-        <p clas="govuk-body">@body</p>
+        <p clas="govuk-body">@Html.Raw(body)</p>
```

Comment:
change the object type to HtmlString instead of using raw

https://stackoverflow.com/questions/2293357/what-is-an-mvchtmlstring-and-when-should-i-use-it#2293662

reject

Reply:
dealt with by making html clearer in tempdata key name